src/core/api/public_api.ts

```diff
@@ -221,6 +221,8 @@ interface IPublicAPIEvent {
   streamEvent : IStreamEvent;
   streamEventSkip : IStreamEvent;
   inbandEvents : IInbandEvent[];
+  trickModeStarts : undefined;
+  trickModeStops : undefined;
```

Comment:
Are event names supposed to be conjugated as `trickModeStarts` or should they be more like `trickModeStart`?

I have a doubt but a feeling that it's generally the latter